### PR TITLE
armv5 rust version frozen for no-atomic support

### PR DESCRIPTION
**Pull request by th0ma7**
## Description

armv5 rust version frozen for no-atomic support

Fixes # 

## Checklist

- [ ] Build rule `all-supported` completed successfully
- [ ] New installation of package completed successfully
- [ ] Package upgrade completed successfully (Manually install the package again)
- [ ] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [ ] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [x] Bug fix
- [ ] New Package
- [ ] Package update
- [x] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by th0ma7** [comment]
@hgy59 would you mind giving this a look?  As expected changes where relatively minor for this to work out (or at least seems to be ok at first glance).

**Comment by hgy59** [comment]
@th0ma7 LGTM, alas the build artifacts are strange. Build [6603](https://github.com/SynoCommunity/spksrc/actions/runs/9294008210) shows 12 Artifacts but only one file for "Packages for aarch64-6.2.4" is available.
Similar result for Build [6602](https://github.com/SynoCommunity/spksrc/actions/runs/9292390043) of sabnzbd: 11 Artifacts but only one file for "Packages for comcerto2k-7.1".
Before, build [6601](https://github.com/SynoCommunity/spksrc/actions/runs/9262167048) was correct (11 artifacts for jellyfin).

File: spk/python311/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 SPK_NAME = python311
 SPK_VERS = 3.11.5
 SPK_VERS_MAJOR_MINOR = $(word 1,$(subst ., ,$(SPK_VERS))).$(word 2,$(subst ., ,$(SPK_VERS)))
-SPK_REV = 8
+SPK_REV = 9
```
[the remark this answers is not in the document]
Comment (th0ma7) [reply spk/python311/Makefile:4]: for sure, this was just meant to trigger the build but your eagle eye caught it as usual :)

**Comment by th0ma7** [comment]
> alas the build artifacts are strange. Build [6603](https://github.com/SynoCommunity/spksrc/actions/runs/9294008210) shows 12 Artifacts but only one file for "Packages for aarch64-6.2.4" is available.

Interestingly the aarch64 showed as canceled so I force-run failed builds, which in turn worked out.

> Similar result for Build [6602](https://github.com/SynoCommunity/spksrc/actions/runs/9292390043) of sabnzbd: 11 Artifacts but only one file for "Packages for comcerto2k-7.1". 

Similar again, build was marked as skipped (not failed) so i forced-run it as well.

> Before, build [6601](https://github.com/SynoCommunity/spksrc/actions/runs/9262167048) was correct (11 artifacts for jellyfin).

I would presume it's something else with the github-action that doesn't manage forced-run artifacts correctly whereas older artifacts somehow disapear.

**Pull request merged**